[FAQ] Import watch only xpub wallet (#758)
zkSNACKs/WasabiDoc: docs/FAQ/FAQ-UseWasabi.md

```
:::

:::details
### Can I import a watch-only extended public key?

> Yes, but not yet in the GUI, you will need to create your own wallet file.

Author: ```suggestion
Yes, but not yet in the GUI, you will need to manually create a new wallet file.
```

> Yes, but not yet in the GUI, you will need to manually create a new wallet file.

Reviewer: ```suggestion
Yes, but not yet in the GUI, you will need to manually create a new wallet file.

```

Open a text editor and paste the following wallet structure:

```
{
  "EncryptedSecret": null,
  "ChainCode": null,
  "MasterFingerprint": null,
  "ExtPubKey": "",
  "PasswordVerified": true,
  "MinGapLimit": 21,
  "AccountKeyPath": "84'/0'/0'",
  "BlockchainState": {
    "Network": "Main",
    "Height": "0"
  },
  "HdPubKeys": [
  ]
}
```

> Then paste your extended public key in-between the quotes of the field `"ExtPubKey": "paste xpub here",`.

Reviewer: ```suggestion
Then paste your extended public key in-between the quotes of the field `"ExtPubKey": "paste xpub here",`.

```

> You can also change the `"AccountKeyPath": "84'/0'/0'",` field if you want to import a different derivation path.

Author: Yes.
You can change the last, to for examle `84'/0'/1'`.

But notice that Wasabi only works with native SegWit bech32 addresses.

> Save this file in your [`Wallets` data folder](/FAQ/FAQ-UseWasabi.html#where-can-i-find-the-wasabi-data-folder) with `WalletName.json`.

Reviewer: ```suggestion
Save this file in your [`Wallets` data folder](/FAQ/FAQ-UseWasabi.html#where-can-i-find-the-wasabi-data-folder) as a json file like this: `WalletName.json`.
```

The `WalletName` will be displayed in the GUI.

> Then start Wasabi and load the wallet to synchronize it.

Reviewer: ```suggestion
Then start Wasabi and load the wallet to synchronize it.

```

For watch only wallets, the `Send` tab is disabled.
However, you can use the `Build Transaction` tab in the `Advanced` section of the `Wallet Explorer` to build an unsigned PSBT transaction.

> When this is signed on the device with the private key (like an offline laptop running Electrum wallet), then you can use the `Broadcast Transaction` tab in the `Tools` menu.

Reviewer: ```suggestion
When this is signed on the device with the private key (like an offline laptop running Electrum wallet or a hardware wallet), then you can broadcast the signed transaction using the `Broadcast Transaction` tab in the `Tools` menu.
```

Author: Yes, but for hardware wallets you don't really need to hack around with the new custom wallet file - you just plug it in. 
this is why I did not mention it here.
